src/RealMeService.php

```diff
@@ -467,28 +499,28 @@ public function getAuthData()
             // Process response and capture details
             $auth = $this->getAuth();
 
-            if(!$auth->isAuthenticated()) {
+            if (!$auth->isAuthenticated()) {
                 throw new RealMeException(
```

Comment:
You've renamed this to `SilverStripe\RealMe\Exception` =)

Reply:
`use SilverStripe\RealMe\Exception as RealMeException;` <- Is that frowned upon?

Reply:
Oh true! Ignore me =)